nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -33,13 +33,15 @@ let
     PAPERCLIP_ROOT_PATH = "/var/lib/mastodon/public-system";
     PAPERCLIP_ROOT_URL = "/system";
     ES_ENABLED = if (cfg.elasticsearch.host != null) then "true" else "false";
-    ES_HOST = cfg.elasticsearch.host;
-    ES_PORT = toString(cfg.elasticsearch.port);
+    ES_PRESET = cfg.elasticsearch.preset;
```

Comment:
Could you also make this conditional on `cfg.elasticsearch.host`?

Reply:
Fixed.